src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -187,3 +197,64 @@ def test_unordered_list_match(inputs, outputs):
         assert all(input in outputs for input in inputs)
     except KeyError as exception:
         raise AssertionError("lists do not match") from exception
+
+
+def transform_model(
+    input_model,
+    output_model,
+    property_transform_keys,
+    property_transform,
+    jinja_template,
+):
+    subprocess.getoutput("npm install jsonata")
```

Comment:
This is still re-installing every run, right?
[Wondering if it's possible to install `jsonata` when `cloudformation-cli` itself is installed](https://stackoverflow.com/questions/49953785/which-options-do-exist-for-defining-a-python-package-with-node-js-dependencies)
If not, I think that's still my biggest concern with pulling in `npm` dependencies